htdocs/accountancy/bookkeeping/list.php

```diff
@@ -1,9 +1,10 @@
 <?php
 /* Copyright (C) 2013-2016  Olivier Geffroy         <jeff@jeffinfo.com>
  * Copyright (C) 2013-2016  Florian Henry           <florian.henry@open-concept.pro>
- * Copyright (C) 2013-2022  Alexandre Spangaro      <aspangaro@open-dsi.fr>
+ * Copyright (C) 2013-2022  Open-DSI      			<support@open-dsi.fr>
```

Comment:
For legal reason, the copyright must be a name of a real physical people.
The moral right on code owns to the author, not the company. This this page in french
https://hal.archives-ouvertes.fr/hal-02399517/file/20191202_plaquette_pi_licences_V1.1.pdf

Email is less important and may expire when the author change its job. So for email, you can use a generic email of a company, but name is important and must remain the real author (physical people).

Reply:
@eldy 
I resolved it